python/sqlflow_submitter/xgboost/train.py

```diff
@@ -64,3 +147,21 @@ def train(datasource,
                         **train_params)
         bst.save_model("my_model")
```

Comment:
One more question, does `bst.save_model("my_model")` pull model parameters from rabbit server and save it to local disk?

Reply:
Yes, I have tested it.